docs/get-started.rst

```diff
@@ -99,8 +99,30 @@ with a *PostgreSQL* container.
 
 .. _run-kinto-python:
 
-Using the Python package
-========================
+
+Installing Python 2.7/3.4+ on Ubuntu/Debian
+-------------------------------------------
+
+Install using the following command:-
+
+::
+
+    sudo apt-get install python2.7 python2.7-dev
```

Comment:
You should not mention `2.7` since in the title you wrote `2.7/3.4+`.

python is installed by default on ubuntu/debian.

python-dev is already mentionned below in the system requirements section.